api/src/main/java/org/openmrs/module/fhir2/spring/FhirAutoProxyCreator.java

```diff
@@ -37,92 +38,64 @@
  * module, even though they do not follow our usual pattern of registering with the service
  * container.
  */
-public class FhirAutoProxyCreator extends AbstractAutoProxyCreator {
+@Configuration
+@EnableAspectJAutoProxy
+public class FhirAutoProxyCreator {
```

Comment:
This would probably be better named `FhirSpringConfiguration` or something like that with these changes, right?

Reply:
Changed to FhirAopConfiguration.